packages/web/src/styles/Base.js

```diff
@@ -20,5 +20,4 @@ const Base = styled(({ as: T = 'div', ...props }) => <T {...props} />)`
 		box-sizing: border-box;
 	}
 `;
-
```

Comment:
Let's have this line back 🔙

Reply:
Gotcha! I think my prettier rules got messed u... It should have been caught in the commit pre-hook 🤔